src/Altinn.Platform/Altinn.Platform.Events/Events/Helpers/PlatformHttpException.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Net.Http;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Events.Helpers
```

Comment:
This is the wrong namespace for an exception.

Reply:
moved to .Exceptions namespace